### DISCUSSION TITLE
Site Overhaul: December 2020-May 2021

### DISCUSSION THREAD

**Discussion by Accio1** [theforumhelpers/theforumhelpers.github.io#35] [General]
### Site Overhaul: December 2020-January 2021
---
This discussion thread is for conversation about the site overhaul, which is planned for late December 2020-early January 2020. The project with the plan for this overhaul can be found here: https://github.com/orgs/theforumhelpers/projects/2/

**In this thread, feel free to discuss:**
- Your thoughts on the site overhaul in general (talk about specific features in the corresponding issues)
- Which issues/features should be prioritized
- Any features that you would like to see but don't necessarily need a separate issue.
- Anything else pertaining to the site overhaul

**Site Overhaul Main Feature Overview:**
- Improve quickreply
    - This includes adding more customization, such as inputting certain values into a phrase using dropdowns
    - Possibly adding a way to add your own replies/save replies
    - Improving existing replies to make them more up to date with the values of the Forum Helpers. (eg. make them more informative for users who may be new to the forums)
- Improvements to the list of Forum Helpers page
    - Search filters (eg. sort alphabetically)
        - Possibly integrating with ocular/shefwrld to sort by post count.
    - Counter to show current number of Forum Helpers, as well as how many curators and managers.
- General site improvements
    - Applications open/closed indicator that updates semi automatically
    - This site has had several new contributors recently, so a credits page or way to indicate contributors is needed.
    - Fix styling to make it consistent throughout the site (fonts, colours, etc.)

**Comment by Accio1** [comment]
@leahcimto I'm on break now, so I'm planning to start the site overhaul. You have been fairly active in this repository recently, so are you interested in working on any of the features above? Please let me know by responding to this comment so that I can create issues for the features and assign them to you. If you don't want to work on any of those features, that is fine also.

**Reply by leahcimto** [reply]
@Accio1 I could do the styling.

**Reply by Accio1** [reply]
@leahcimto I am going to create an issue soon with all of the styling issues compiled into one spot. Then I will assign you and you can get started. I want to have time to think about how to make styles consistent and have some discussion about it before you start working on it so that you don't have to redo your work part way through.

**Comment by Accio1** [comment]
Before we start work on any of these features, I am going to create new issue and PR templates so that things can be more organized.

**Reply by leahcimto** [reply]
@Accio1 Ok! When do you think they will be done?

**Reply by Accio1** [reply]
I hope to finish them today

**Comment by leahcimto** [comment]
@Accio1 When will the issues be created?

**Reply by Accio1** [reply]
@leahcimto Feel free to create issues for parts that you want to work on. I will create issues for remaining parts. Don't create an issue for styling as I am working on one that can be discussed before you start on styling.

**Comment by leahcimto** [comment]
The application indicator isn't very important anymore since it seems like they are always open.

**Reply by Accio1** [reply]
+1

**Comment by leahcimto** [comment]
Since this overhaul is getting pretty delayed I changed the title since it is no longer January.

**Comment by leahcimto** [comment]
@Accio1 Hi! I forget if this was on this list at some point, or on the project or whatever, but I remember there might have been something like displaying if the forums were up. If this were to be implemented, I really like upptime.js.org. It's really easy to use. I made a forum uptime [here](https://forumuptime.leahcimto.com/), and think something like this could be implemented (on its own page, perhaps).

**Reply by FunctionalMetatable** [reply]
+1 upptimes are good
